src/app/helpers.js

```diff
@@ -42,10 +43,10 @@ function getStatusStyle(status, property) {
 // Skip team name if `skipTeam` is true.
 // Skip `prefix |` if `prefix` empty.
 function pageTitle(prefix, skipTeam, team) {
-  let suffix = 'Check';
+  let suffix = capitalize(config.appName);
```

Comment:
Still used on src/app/components/project/Project.js line 63, in a redundant way... let's change it.